src/stable_hasher.rs

```diff
@@ -85,23 +166,23 @@ impl StableHasher {
     /// To be used in-place of [`Hasher::finish`].
```

Comment:
This comment still mentions `StableHasherResult`

Reply:
My bad, fixed.